todo_app/test/integration/test_client.py

```diff
@@ -0,0 +1,127 @@
+import json
+import os
+from dotenv import load_dotenv, find_dotenv
+import pytest
+import requests
+from todo_app import app
+from mock import Mock, ANY
```

Comment:
This line errors for me, I needed to either:
1. Import via unittest:
```suggestion
from unittest.mock import Mock, ANY
```
2. Add `mock` to my project dependencies (not `pytest-mock`)

If you don't see that, the most likely cause I'd guess is that you're running the tests through the VSCode interface, and VSCode is using an environment different to this project's virtual environment (e.g. it's using your global environment). When you've got a python file open, check that the version of python listed in the very bottom right mentions that it's a `.venv` environment (e.g. mine says `3.11 ('.venv': poetry)`) and then hopefully you'll be able to repro

If you go with the import change mentioned above, we should be equally able to remove the `pytest-mock` library

Reply:
The version in VSCode was fine (mine says `3.11.5 (.venv): Poetry`), it also worked fine when I ran the tests in the terminal using `poetry run pytest` or just `pytest`. I think this happened because I maybe had mock installed with pip? I started getting the import error after I ran `pip uninstall mock`

I replaced the poetry dependency, hopefully it works for you now?

Reply:
Hmm, it _shouldn't_ be falling back to the pip-installed library when run through poetry, but plausibly! Certainly that would explain it passing with just `pytest` though

Definitely working now in any case 😄